pkg/cmd/step/create/pr/step_create_pr_regex.go

```diff
@@ -80,7 +80,7 @@ func NewCmdStepCreatePullRequestRegex(commonOpts *opts.CommonOptions) *cobra.Com
 		},
 	}
 	AddStepCreatePrFlags(cmd, &options.StepCreatePrOptions)
-	cmd.Flags().StringVarP(&options.Regexp, "regex", "", "", "The regex to use when doing updates")
+	cmd.Flags().StringArrayVarP(&options.Regexps, "regexs", "", make([]string, 0), "The regex to use when doing updates")
```

Comment:
should this stay as `regex` to make the flag backwardly compatible?

Reply:
Ah, yup, I misunderstood how the array var names worked. Fixing.